src/Datasource/EntityInterface.php

```diff
@@ -95,9 +95,14 @@ public function visibleProperties();
      * Returns an array with all the properties that have been set
      * to this entity
      *
+     * ### Options
+     *
+     * * `includeHidden` Include hidden properties in array
+     *
+     * @param array $options List of options
      * @return array
      */
-    public function toArray();
+    public function toArray($options = []);
```

Comment:
PS : It is not specified that toArray do not contains hidden. It's hidden but it's set.
```
     * Returns an array with all the properties that have been set
     * to this entity
```
It's specified in the $_hidden field :
```
* List of property names that should **not** be included in JSON or Array
* representations of this Entity.
```